setup.py

```diff
@@ -40,8 +40,8 @@ def runSetup():
                 'psutil==3.0.1'],
             'aws': [
                 botoRequirement,
-                'cgcloud-lib==' + version.cgcloudVersion,
-                'futures==3.0.5'],
+                'futures==3.0.5',
+                'pycrypto==2.6.1'],
```

Comment:
Hi!

pycrypto hasn't been maintained in 3 years. It also doesn't support Python 3. (I think this last point is a work in progress for toil)

 Consider using [pycryptodome](https://pypi.python.org/pypi/pycryptodome) (drop-in replacement) or [cryptography](https://pypi.python.org/pypi/cryptography) as a dependency.

Reply:
I created a separate issue for this. Thanks @psafont.